Grid/Grid.php

```diff
@@ -632,7 +633,7 @@ protected function processMassActions($actionId)
                         $action->getParameters()
                     );
 
-                    $subRequest = $this->container->get('request')->duplicate([], null, $path);
+                    $subRequest = $this->request->duplicate(array(), null, $path);
```

Comment:
Don't know about best practices on that, sry.
As ```$this->request``` was used everywhere and ```$this->container->get``` only here, I choosed the first option.